src/server/api/controllers/users.controller.js

```diff
@@ -8,14 +9,40 @@ const {
 
 const { isInteger } = require('../lib/utils/validators');
 
-const getUsers = () => {
-  return knex('users').select(
-    'users.id',
-    'users.fullName',
-    'users.email',
-    'users.createdOn',
-    'users.firebaseUId',
-  );
+const getUsers = async () => {
+  const result = await client.search({
+    index: usersIndex,
+    body: {
+      query: {
+        match_all: {},
+      },
+      size: 50,
+      // sort: ['fullName.keyword'],
```

Comment:
fullName is now a keyword and it doesn't have a subfield named keyword; that's why this didn't work.